dio/lib/src/dio/dio_for_browser.dart

```diff
@@ -1,10 +1,8 @@
+import 'dart:async';
+import 'dart:html';
+
+import '../../dio.dart';
```

Comment:
Do not import the export library.

Reply:
```dart
import 'dart:async';
import 'dart:html';
```

These packages are needed to create a blob file !

Reply:
I mean `../../dio.dart`.

Reply:
changed